float32 x
float32 y
float32 cor_x
float32 cor_y
float32 cor_z
float32 size
float32 angle
float32 response
int32 octave
int32 class_id
float32[] descriptor
float32 rating
